akka-actor/src/main/scala/akka/routing/OptimalSizeExploringResizer.scala

```diff
@@ -125,7 +125,7 @@ case class DefaultOptimalSizeExploringResizer(
     numOfAdjacentSizesToConsiderDuringOptimization: Int = 16,
     exploreStepSize: Double = 0.1,
     downsizeRatio: Double = 0.8,
-    downsizeAfterUnderutilizedFor: Duration = 72.hours,
+    downsizeAfterUnderutilizedFor: FiniteDuration = 72.hours,
```

Comment:
This is a breaking (not bin compatible) change. We have to keep Duration here, but can add a val inside the class:
```
private val downsizeAfterUnderutilizedDuration = downsizeAfterUnderutilizedFor.asInstanceOf[FiniteDuration].toJava
```